nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/pom.xml

```diff
@@ -45,6 +45,17 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Can we add azure-security-keyvault-secrets to the `<dependencyManagement/>` section of the root `nifi-azure-bundle/pom.xml` and just inherit the version from there (as a property)?